lib/r10k/cli/version.rb

```diff
@@ -12,7 +12,16 @@ def self.command
         summary 'Print the version of r10k'
 
         run do |opts, args, cmd|
-          puts R10K::VERSION
+          puts "r10k #{R10K::VERSION}"
+          if opts[:verbose]
+            puts RUBY_DESCRIPTION
+            cmdpath = caller.last.match(/\A(.*#{$PROGRAM_NAME}):/).captures.first
+            puts "Command path: #{cmdpath}"
+            puts "Interpreter path: #{Gem.ruby}"
+            if Gem::Version.new(RUBY_VERSION.dup) >= Gem::Version.new('1.9.3')
```

Comment:
If we're running on Ruby 1.8.7 we don't have a guarantee that rubygems was loaded; I would prefer that we do a naive string compare (`RUBY_VERSION >= '1.9.3'`) and skip the `Gem::Version`.